src/tools/pkcs11-tool.c

```diff
@@ -272,6 +273,7 @@ static const struct option options[] = {
 	{ "private",		0, NULL,		OPT_PRIVATE },
 	{ "sensitive",		0, NULL,		OPT_SENSITIVE },
 	{ "extractable",	0, NULL,		OPT_EXTRACTABLE },
+	{ "undestroyable",	0, NULL,		OPT_UNDESTROYABLE },
```

Comment:
This is missing a documentation in the manual page for the pkcs11-tool (hence the CI failure)

Reply:
Good catch!
Sorry for the mistake.

Reply:
I meant the manual page in `doc/tools/pkcs11-tool.1.xml`.